src/temaWeek6LocalStore/main/LocalStore.java

```diff
@@ -0,0 +1,258 @@
+	
+	
+	//	menu
+	public void getMenu() {
+		boolean running = true;
+		
+		System.out.println("Welcome to LocalStore!");
+		while (running) {
+			System.out.println("\n Enter your choice: " +//options in the menu
+					"\n 1.Create product and add it to stock" +
+					"\n 2.Sell products" +
+					"\n 3.Display daily sales report" +
+					"\n 4.Exit");
+			
+			
+			Scanner scan = new Scanner(System.in);
+			String option = scan.next();
+			LocalStore lc = new LocalStore();
+			switch (option) {
+				case "1": {
+					System.out.println("Product type (1 - animal; 2 - vegetable):");
+					String type = scan.next();
+					if (type.equals("1")) {//adds an animal product to stockItem
+						
+						lc.addToStockAnimal(stockItems);
+						break;
+					}
+					if (type.equals("2")) {//adds an vegetable product to stockItem
+						lc.addToStockVegetable(stockItems);
+						break;
+					}
+					if(!type.equals("1")||!type.equals("2")){
+						System.out.println("Enter 1 or 2!");
+						break;
+					}
+				}
+				case "2": {//sell products based on input
+					System.out.println("Items in stock: \n"+stockItems);
+					System.out.println("Enter ID ");
+					String inputID = scan.next();
+					lc.setInputQuantity(inputQuantity);
+					inputQuantity=lc.getInputQuantity();
+					lc.addSoldItems( stockItems,soldItems,inputID,inputQuantity);//adds sold items to soldItem list
+					lc.sellProduct(stockItems,inputID,inputQuantity);
+					//updates the list of stockItems based on input
+					System.out.println("");
+					System.out.println("Remaining items in stock: \n"+stockItems);//print remaining items in the stockList
+					break;
+					
+				}
+				case "3": {
+					System.out.println("");
+					printSoldItems(soldItems);//print soldItem list
+					break;
+				}
+				case "4": {
+					running = false;
+				}
+			}
+			
+		}
+		
+	}
+	//	method created a stockItem of type animal
+	public StockItem createAnimalProduct() {
+		Animal a=new Animal(name,price, validityDate,weight, ID,storageTemp);
+
+//		calling name method
+		name=a.setName(name);
+
+//		calling setPrice method
+		price=a.setPrice(price);
+
+//		calling setValidityDate method
+		validityDate=a.setValidityDate(validityDate);
+
+//		calling setWeight method
+		weight=a.setWeight(weight);
+
+//		calling setID method
+		ID=a.setID(ID);
+		
+		//		calling setStorageTemperature
+		storageTemp=a.setStorageTemp(storageTemp);
+
+//		calling quantity method form StockItem
+		quantity=this.stock.setQuantity();
+		Product p=new Product(name,price, validityDate,weight, ID);
+		
+		Animal a1=new Animal(name,price, validityDate,weight, ID,storageTemp);
+		p=a1;
+		return new StockItem(a1,quantity);
+	}
+	
+	//method creates a vegetable product and adds the item to the list of stockItems
+	public List<StockItem> addToStockAnimal(List<StockItem> stockItems){
+		StockItem s=createAnimalProduct();
+		stockItems.add(s);
+		return stockItems;
```

Comment:
without parameter or return type you could directly modify the stockItems attribute on this object